proof/refine/AARCH64/ArchCSpace1_R.thy

```diff
@@ -228,51 +248,45 @@ qed
 
 lemmas setCTE_typ_ats[wp] = typ_at_lifts[OF setCTE_typ_at']
 
+context
+begin
```

Comment:
I have a vague feeling I'd want a comment here to indicate the purpose of the context, or maybe at the end, but it's hard to say what would help exactly. I guess if it's this short, we can leave it unadorned, but I never know what to do with these things.

Reply:
Yup, I can buy that. Adding a comment to the `end` like you did in the other situation I applauded would work too.